docs/source/specification.md

```diff
@@ -22,102 +27,147 @@ For example:
         └── ...
 ```
 
-The project folder may have any name descriptive of the project (that does not include spaces). Within the 
-project folder, data is separated into `rawdata` and `derivatives`. `rawdata` contains the raw
-data acquired during data collection. `derivatives` includes any processed data (e.g. spike sorting or pose estimation). 
-The subject / session / datatype folder structure within derivatives should
-match the rawdata folder whenever possible.
+### Basic principles
+
+* The project folder ***may*** have any name descriptive of the project but this ***must not*** contain spaces.
+
+* Within the project folder, data is separated into `rawdata` and `derivatives`. 
+
+  * `rawdata`: data coming out of the data acquisition system (e.g. binary files, tiffs, videos files). 
+
+  * `derivatives`: any processed data that is derived from `rawdata` (e.g. spike sorting or pose estimation).
 
-Each subject (e.g. mouse, rat) in the project has a dedicated folder in which all data from 
-experimental sessions are stored.
-In the case of multiple sessions, each session has its own folder where acquired data, belonging to
-a dataype (e.g. `ephys, behav, funcimg, histology`) are stored.
+* Data within the `rawdata` folder ***must*** be hierarchically structured into `subject/session/datatype` levels. Each level ***must*** contain at least one folder corresponding to the next (lower) level.
```

Comment:
maybe an example here would be useful off the bat (e.g. `sub-001/ses-001/ephys`). [oh I forgot there is an example dir tree at the top, maybe this is not required]